portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,15 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+    const randomQuote =
+        ['Chaos is Ladder.', 'I am a native Spanish speaker.', 'I enjoy learning about history.',
+            'I was born and raised in El Paso, Texas.'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Pick a random greeting.
+    const quote = randomQuote[Math.floor(Math.random() * randomQuote.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    // Add it to the page.
+    const quoteContainer = document.getElementById('quote-container');
+    quoteContainer.innerText = quote;
```

Comment:
Nice! This is the kind of pattern I was referring to above with the button to add a random quote.